docs/stacks-guide-rook.md

```diff
@@ -0,0 +1,874 @@
+---
+title: "Stacks Guide: Rook" toc: true weight: 550 indent: true
+---
+
+# Stacks Guide: Rook
+
+> Note: this guide does not follow along with the other stacks guides. It
+> deploys a different application (Yugastore) and involves using the GCP stack
+> along with the Rook stack.
+
+This user guide will walk you through [Yugastore] application deployment using
+Crossplane's [Rook] stack to run [YugabyteDB] in a Google Cloud [GKE] Kubernetes
+cluster. To do so, we will broadly:
+
+1. Provision a GKE Kubernetes cluster
+1. Install the Rook [Yugabyte operator] into the GKE cluster
+1. Provision a YugabyteDB cluster and deploy the Yugastore app into the GKE
+   cluster
+
+... all using Crossplane!
+
+## Table of Contents
+
+1. [Pre-requisites](#pre-requisites)
+2. [Preparation](#preparation)
+3. [Set Up Resource Classes](#set-up-resource-classes)
+4. [Provision GKE Cluster and Install Rook Yugabyte
+   Operator](#provision-gke-cluster-and-install-rook-yugabyte-operator)
+5. [Deploy Yugastore alongside
+   YugabyteDB](#deploy-yugastore-alongside-yugabytedb)
+6. [Cleanup](#cleanup)
+7. [Conclusion and Next Steps](#conclusion-and-next-steps)
+
+1. [Pre-requisites](#pre-requisites)
+1. [Preparation](#preparation)
+1. [Set Up Crossplane](#set-up-crossplane)
+    1. [Install in Standalone Control Cluster](#install-in-standalone-control-cluster)
+    1. [GCP Provider](#gcp-provider)
+    1. [Rook Provider](#rook-provider)
+    1. [Resource Classes](#resource-classes)
+1. [Provision Target Cluster](#provision-target-cluster)
+    1. [Provision GKE Cluster with a Resource Claim](#provision-gke-cluster-with-a-resource-claim)
+    1. [Install Rook Yugabyte Operator with a `KubernetesApplication`](#install-rook-yugabyte-operatore-with-a-kubernetesapplication)
```

Comment:
link is broken: needs "operatore" -> "operator"

also for the heading and ToC consider using just plain text, instead of inline code markdown for `KubernetesApplication` -- mainly for visual / consistency reasons, e.g.

![image](https://user-images.githubusercontent.com/284840/67623504-ad10c780-f7da-11e9-80ea-fd26e18b5728.png)